src/styles/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 200px;
+  height: 408px;
```

Comment:
Try don't hardcode height.
Height should depend from content

Reply:
> Try don't hardcode height. Height should depend from content

Not fixed